### PR TITLE
Update UI test buildspec to run the new tests + remove old UI Tests

### PR DESCRIPTION
**Pull request by hunterwerlla**
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -103,11 +103,13 @@ To test your changes locally, you can run the project from IntelliJ or gradle.
   - Requires `sam` CLI to be on your `$PATH`.
 - To run **GUI tests**:
   ```
-  ./gradlew guiTest
+  ./gradlew uiTestCore
   ```
   - To debug GUI tests,
     1. Set `runIde.debugOptions.enabled=true` in the gradle file.
```
Comment (abrooksv) [inline CONTRIBUTING.md:109]: Is this right? It also probably easier to just say run `./gradlew runIdeForUiTests --debug-jvm`

File: ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/IdeaFrame.kt [hunk 2]
```diff
@@ -92,10 +92,10 @@ class IdeaFrame(remoteRobot: RemoteRobot, remoteComponent: RemoteComponent) : Co
         }
     }
 
-    // Tips sometimes open when running locally, close it if it opens
+    // Tips sometimes open when running, close it if it opens
     fun tryCloseTips() {
```
Comment (abrooksv) [inline ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/IdeaFrame.kt:96]: Can we kill this now?

**Comment by abrooksv** [comment]
What do we want to do about coverage? Do we want to attach the Jacoco agent to the runIde target?

File: ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/IdeaFrame.kt (hunk 2 above)
Comment (hunterwerlla) [reply ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/IdeaFrame.kt:96]: Nope, these show up on the tests running on codebuild

Comment (abrooksv) [reply ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/IdeaFrame.kt:96]: Oh right...d'oh thats 203..

File: CONTRIBUTING.md (hunk 1 above)
Comment (hunterwerlla) [reply CONTRIBUTING.md:109]: This is correct, and yep that is easier, changed

**Comment by hunterwerlla** [comment]
> What do we want to do about coverage? Do we want to attach the Jacoco agent to the runIde target?

We already do that: `jacoco.applyTo(runIdeForUiTests)`

**Comment by abrooksv** [comment]
> > What do we want to do about coverage? Do we want to attach the Jacoco agent to the runIde target?
> 
> We already do that: `jacoco.applyTo(runIdeForUiTests)`

https://github.com/aws/aws-toolkit-jetbrains/blame/a6b8188ff4a681b1cfc5c2baca82a091a2e24bf0/build.gradle#L196

Yikes....I made that change.... I can't keep track anymore...

File: ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/WelcomeFrame.kt [hunk 3]
```diff
@@ -40,6 +40,7 @@ class WelcomeFrame(remoteRobot: RemoteRobot, remoteComponent: RemoteComponent) :
             actionLink("Open").click()
         }
         fillSingleTextField(path.toAbsolutePath().toString())
```
Comment (abrooksv) [inline ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/WelcomeFrame.kt:42]: Can we hide this in a fixture? I presume we would have this logic followed by next line anytime we need to browse for files (i.e. S3 upload/download)

Comment (hunterwerlla) [reply ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/WelcomeFrame.kt]: Yep, replaced

**Pull request merged**